src/thermompnn/proteinmpnn/model_utils.py

```
@@ -0,0 +1,1175 @@
+import itertools
```

Ruff (0.8.2)

`itertools` imported but unused

Remove unused import: `itertools`

(F401)